conftest.py

```diff
@@ -1,28 +1,45 @@
 """Setup for Pytest."""
 
 # Standard library imports
+from pathlib import Path
+import shutil
+from sys import platform
 import warnings
 import webbrowser
 
 # Third party imports
 import pytest
-
+from PIL import Image, ImageChops
 
 # ---- Constants
 
 OPEN_BROWSER_OPTION = '--open-browser'
+COMPARE_SCREENSHOTS_OPTION = "--compare-screenshots"
+UPDATE_REFERENCE_SCREENSHOTS_OPTION = "--update-reference-screenshots"
```

Comment:
```suggestion
COMPARE_SCREENSHOTS_OPTION = '--compare-screenshots'
UPDATE_REFERENCE_SCREENSHOTS_OPTION = '--update-reference-screenshots'
```

Consistent string quotes (I normalized the handful that were inconsistent in #40 )